docs/user-guide/cli-using-sharing-team-config-files.md

```diff
@@ -0,0 +1,98 @@
+# Sharing team configuration files
+
+Team leaders can share team configuration files using several methods:
+- Shared network drive
+- Project repository (for example, GitHub)
+- Web server
+
+The following topics describe how to share the team configuration files.
+
+## Network drive
+
+1. Store the configuration files on a shared network drive.
+
+2. Issue the following command:
+
+    ```
+    zowe config import <DriveLetter>:\<SharedZoweConfig>\zowe.config.json
+    ```
+    - DriveLetter:
+
+        The drive letter of the shared network drive
+
+    - SharedZoweConfig:
+
+        The directory path on the drive
+
+    **Note:** You can specify any path that Windows Explorer can access. For example, a UNC network path (`\\<HostName>\SharedZoweConfig\zowe.config.json`) or local file path (`C:\Users\<UserName>\Downloads\zowe.config.json`).
```

Comment:
Sorry I think I mentioned Windows Explorer here, but we should probably use a more generic cross-platform term. Maybe "your file manager" instead of "Windows Explorer". Or "your file manager (e.g., Windows Explorer)" if we want to keep mentioning it.